tests/unit/lms/views/helpers/_via_test.py

```diff
@@ -20,7 +20,13 @@ def test_if_creates_the_correct_via_url(self, pyramid_request):
 
         assert final_url == Any.url.matching(
             "http://test_via3_server.is/route"
-        ).with_query(dict(self.DEFAULT_OPTIONS, url=url))
+        ).with_query(
+            dict(
+                self.DEFAULT_OPTIONS,
+                url=url,
+                sec="eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1cmwiOiJodHRwOi8vVEVTVF9WSUEzX1NFUlZFUi5pcy9yb3V0ZT91cmw9aHR0cCUzQSUyRiUyRmV4YW1wbGUuY29tIiwiZXhwIjoxNjEyNzc4NDAwfQ.X1ydoVIX59VZivCdswfe65t8by1k75RJYUI-hE_EC9k",
```

Comment:
I think we might want to vague this up a bit, unless we care about the specific value or implementation for some reason.

`Any.string()`?